scripts/install

```diff
@@ -14,110 +8,96 @@ source /usr/share/yunohost/helpers
 	set -o nounset
 
-	ynh_exec_warn_less ynh_exec_as $app RUSTUP_HOME="$install_dir"/.rustup CARGO_HOME="$install_dir"/.cargo bash -c 'curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -q -y  --default-toolchain=stable --profile=minimal'
+	ynh_hide_warnings ynh_exec_as_app RUSTUP_HOME="$install_dir"/.rustup CARGO_HOME="$install_dir"/.cargo bash -c 'curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -q -y  --default-toolchain=stable --profile=minimal'
 	export PATH="$install_dir/.cargo/bin:$PATH"
-	ynh_exec_warn_less ynh_exec_as $app env PATH=$PATH pip install --upgrade pip
-	ynh_exec_warn_less ynh_exec_as $app env PATH=$PATH pip install --upgrade 'setuptools<71'
-	ynh_exec_warn_less ynh_exec_as $app env PATH=$PATH pip install woob simplejson BeautifulSoup4 PyExecJS typing-extensions pdfminer.six Pillow chompjs --ignore-installed
-	ynh_secure_remove --file="$install_dir/.cargo"
-	ynh_secure_remove --file="$install_dir/.rustup"
-	ynh_secure_remove --file="$install_dir/.cache"
-	ynh_secure_remove --file="$install_dir/.local"
+	ynh_hide_warnings ynh_exec_as_app PATH=$PATH pip install --upgrade pip
+	ynh_hide_warnings ynh_exec_as_app PATH=$PATH pip install --upgrade 'setuptools<71'
+	ynh_hide_warnings ynh_exec_as_app PATH=$PATH pip install woob simplejson BeautifulSoup4 PyExecJS typing-extensions pdfminer.six Pillow chompjs --ignore-installed
+	ynh_safe_rm "$install_dir/.cargo"
+	ynh_safe_rm "$install_dir/.rustup"
+	ynh_safe_rm "$install_dir/.cache"
+	ynh_safe_rm "$install_dir/.local"
 )
 
 #=================================================
 # INSTALL KRESUS WITH NPM
 #=================================================
-ynh_script_progression --message="Installing $app..." --weight=1
+ynh_script_progression "Installing $app..."
 
 pushd $install_dir
- 	ynh_use_nodejs
-	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH yarn config set network-timeout 300000
-	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH yarn install --production
-	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH yarn cache clean --all
-	ynh_secure_remove --file="$install_dir/.cache"
-	ynh_secure_remove --file="$install_dir/.yarn"
+	ynh_hide_warnings ynh_exec_as_app yarn config set network-timeout 300000
+	ynh_hide_warnings ynh_exec_as_app yarn install --production
+	ynh_hide_warnings ynh_exec_as_app yarn cache clean --all
+	ynh_safe_rm "$install_dir/.cache"
+	ynh_safe_rm "$install_dir/.yarn"
 popd
 
 #=================================================
 # ADD A CONFIGURATION
 #=================================================
-ynh_script_progression --message="Adding a configuration file..." --weight=1
-
-ynh_add_config --template="../conf/config.ini" --destination="$install_dir/config.ini"
+ynh_script_progression "Adding $app's configuration..."
 
-chmod 400 "$install_dir/config.ini"
-chown $app:$app "$install_dir/config.ini"
+ynh_config_add --template="config.ini" --destination="$install_dir/config.ini"
```

Comment:
Same warning about  config.ini path

Reply:
`#REMOVEME? Assuming the file is setup using ynh_config_add, the proper chmod/chowns are now already applied and it shouldn't be necessary to tweak perms | chmod 400 "$install_dir/config.ini"`

```
chmod 400
chown $app:
```
should be applied by the helper 2.1. If the app is writing to the config file then `chmod 600` should be added to the script...

https://github.com/YunoHost/yunohost/blob/5d15c00d921927825a0bf98b0c5d872dac57d1b7/helpers/helpers.v2.1.d/templating#L44

[_ynh_apply_default_permissions() {](https://github.com/YunoHost/yunohost/blob/c86477048375114f0ed62f5052e66faec6723022/helpers/helpers.v1.d/utils#L367)

Reply:
My warning was regarding the config.ini filepath (moving from `../config.ini` to `config.ini`).

Reply:
you mean from `--template="../conf/config.ini"` to `--template="config.ini"`?

Reply:
Yes

Reply:
`../conf/` path is implied, therefore so no need to put the full path